packages/reference/src/assets/WrappedAssetCard/FetchingWrappedAssetCard.tsx

```diff
@@ -128,7 +113,7 @@ export function FetchingWrappedAssetCard(props: FetchingWrappedAssetCardProps) {
       const builtinCardProps: Omit<WrappedAssetCardProps, 'isClickable'> = {
         ...commonProps,
         ...props,
-        getEntityScheduledActions: loadEntityScheduledActions,
+        getEntityScheduledActions: sdk.space.getEntityScheduledActions,
```

Comment:
Originally was using the new `useEntityLoader` hook here but it sends invalid requests, looking for environment id

![Screenshot 2022-08-16 at 20 58 57](https://user-images.githubusercontent.com/109096340/184974515-c42718e4-691f-457c-a731-4c6821c737ff.png)

Reply:
Same in `FetchingWrappedEntryCard`